app/views/foreman/job_templates/change_content_source.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Change content source
+job_category: Katello
+model: JobTemplate
+provider_type: SSH
+description_format: Configure subscription manager to new content source
+feature: katello_change_content_source
+template_inputs:
+- name: Smart Proxy
+  required: true
+  input_type: user
+  value_type: resource
+  resource_type: SmartProxy
+  description: New content source
+  advanced: false
```

Comment:
I still insist we need a template without input sources and only use `host.content_source`.

Reply:
What will happen if `@host` doesn't have any `content_source`? Where in the UI user can set the `content_source` for the host? I can't find it anywhere

Reply:
> What will happen if `@host` doesn't have any `content_source`?

That's a good point. I would check if the content URL and the RHSM URL are non-empty or exit with a failure. Either in the template or on the host. So either `raise Exception unless url` or `if [ -n "$url" ] ; then echo "No URL" ; exit 1 ; fi`.

The alternative would be to unconfigure subman but that feels illogical to me at this point. And it's a template so users can still modify it if they want to implement that.

> Where in the UI user can set the `content_source` for the host? I can't find it anywhere

In the Edit Host form on the `Host` tab there a `Content Source` field. That should configure it.